spec/models/case_contact_report_spec.rb

```diff
@@ -78,166 +78,204 @@
-          create(:case_contact, {casa_case: case_case_2})
-          report = CaseContactReport.new({has_transitioned: true})
-          contacts = report.case_contacts
-          expect(contacts.length).to eq(1)
-        end
+    describe "has transitioned behavior" do
+      it "returns only case contacts the youth has transitioned" do
+        case_case_1 = create(:casa_case, transition_aged_youth: false)
+        case_case_2 = create(:casa_case, transition_aged_youth: true)
+        create(:case_contact, {casa_case: case_case_1})
+        create(:case_contact, {casa_case: case_case_2})
+        report = CaseContactReport.new({has_transitioned: false})
+        contacts = report.case_contacts
+        expect(contacts.length).to eq(1)
+      end
 
-        it "returns case contacts with both youth has transitioned and youth has not transitioned" do
-          case_case_1 = create(:casa_case, transition_aged_youth: false)
-          case_case_2 = create(:casa_case, transition_aged_youth: true)
-          create(:case_contact, {casa_case: case_case_1})
-          create(:case_contact, {casa_case: case_case_2})
-          report = CaseContactReport.new({has_transitioned: ""})
-          contacts = report.case_contacts
-          expect(contacts.length).to eq(2)
-        end
+      it "returns only case contacts the youth has transitioned" do
+        case_case_1 = create(:casa_case, transition_aged_youth: false)
+        case_case_2 = create(:casa_case, transition_aged_youth: true)
+        create(:case_contact, {casa_case: case_case_1})
+        create(:case_contact, {casa_case: case_case_2})
+        report = CaseContactReport.new({has_transitioned: true})
+        contacts = report.case_contacts
+        expect(contacts.length).to eq(1)
       end
 
-      describe "wanting driving reimbursement functionality" do
-        it "returns only contacts that want reimbursement" do
-          # want_driving_reimbursement
-          create(:case_contact, {miles_driven: 50, want_driving_reimbursement: true})
-          create(:case_contact, {miles_driven: 50, want_driving_reimbursement: false})
+      it "returns case contacts with both youth has transitioned and youth has not transitioned" do
+        case_case_1 = create(:casa_case, transition_aged_youth: false)
+        case_case_2 = create(:casa_case, transition_aged_youth: true)
+        create(:case_contact, {casa_case: case_case_1})
+        create(:case_contact, {casa_case: case_case_2})
+        report = CaseContactReport.new({has_transitioned: ""})
+        contacts = report.case_contacts
+        expect(contacts.length).to eq(2)
+      end
+    end
 
-          report = CaseContactReport.new({want_driving_reimbursement: true})
-          contacts = report.case_contacts
-          expect(contacts.length).to eq(1)
-        end
+    describe "wanting driving reimbursement functionality" do
+      it "returns only contacts that want reimbursement" do
+        # want_driving_reimbursement
```

Comment:
This comment is not needed

Reply:
> In test file `app/models/case_contact.rb`, move other test examples out of "occured at range filter" text example. So that, it makes more sense when looking from high level view. See screenshot:

Good find. That bit is unnecessary.
I think it was there before, cos' I just worked on the "contact type group filter functionality" and indented the rest.

Reply:
I didnt realize that reindenting code would create this situation. When I used `git diff` on VSCode, it just showed the code I added and ignore reindentation. 😅